app/services/imports/user_import_service.rb

```diff
@@ -45,7 +45,7 @@ def create_user(xml_document)
     end
 
     def user_field_value(xml_document, field)
-      field_value(xml_document, "user", field)
+      field_value(xml_document, "user", field, { "user" => "dclg:user" })
```

Comment:
I wonder if dclg should be changed to dluhc to be up to date, or if that's something we should be aware might need to happen in the future? Not a major issue I appreciate :)

Reply:
Given the inflexibility of the source I doubt it will change!